src/track/text-style.js

```diff
@@ -173,7 +173,7 @@ class TextStyle {
    */
   fontSize: string = '100%';
 
-  fontScale: number = 1;
+  fontScale: number = 0;
```

Comment:
This is becuase of the "+1" of implicitFontScale?
Please add jsdoc expaining this please and fix the jsdoc of fontSize